app/javascript/components/upload/form-components.jsx

```diff
@@ -90,6 +98,13 @@ export function SaveDeleteButtons({ file, updateFile, saveFile, deleteFile, vali
     </button>
   }
 
+  let deleteText
+  if (file.remote_location) {
+    deleteText = ' will remain in the bucket because you provided a remote path to an existing file.'
+  } else {
+    deleteText = ' will be removed from the bucket.'
```

Comment:
```suggestion
    deleteText = 'remain in the bucket because you provided a remote path to an existing file.'
  } else {
    deleteText = 'be removed from the bucket.'
```